exec-analysis/src/main/java/com/capitalone/dashboard/exec/collector/DefaultDataCollector.java

```diff
@@ -35,7 +39,28 @@
     public Map<String, List<Row>> collectAll() {
         Map<String, List<Row>> rowMap = new HashMap<>();
         DataFrameLoader.loadDataFrame(collectionName, javaSparkContext);
-        Dataset<Row> dataRows = sparkSession.sql(query);
+        Dataset<Row> dataRows = null;
+        String tempQuery = query;
+        if (collectionName.contains("pipelines")) {
+            String[] str = {"Production", "prod"};
+            try {
+                for (int i = 0; i < str.length; i++) {
```

Comment:
Can you update code for the looping here?

Reply:
It's already in a loop, it substitute environments  and pass those to spark query.

Reply:
I mean to say use cleaner looping here.

Reply:
Refactor by calling method and used for each loop